extensions/MESA/EGL_MESA_config_select_group.txt

```diff
@@ -0,0 +1,108 @@
+Name
+
+    EGL_MESA_config_select_group
+
+Name Strings
+
+    EGL_MESA_config_select_group
+
+Contributors
+
+    Hal Gentz <zegentzy@protonmail.com>
+    Adam Jackson <ajax@redhat.com>
+    Robert Mader <robert.mader@posteo.de>
+
+Contacts
+
+    Hal Gentz <zegentzy@protonmail.com>
+
+Status
+
+    Draft
+
+Version
+
+    Version 3, 2020-04-06
+
+Number
+
+    EGL Extension #143
+
+Extension Type
+
+    EGL client extension
+
+Dependencies
+
+    This extension is written against the wording of the 2014.08.27 revision
+    of the EGL 1.5 Specification.
+
+Overview
+
+    This extension provides a mechanism to lower the config selection priority
+    of the configs returned by eglChooseConfig. A new config attribute is
+    introduced, providing a way for the implementation to group configs into
+    different config selection categories.  The config selection priorities of
+    all the configs in one category may be higher or lower than the ones in
+    other categories.  The config selection priorities in one category follow
+    the rules of eglChooseConfig.
```

Comment:
It's probably worth explicitly referencing the GLX extension in this extension.  Possibly just add something here like

```
    See also GLX_SGIX_visual_select_group.
```

Reply:
Done